src/main/groovy/edu/oregonstate/mist/api/Resource.groovy

```diff
@@ -148,6 +148,27 @@ abstract class Resource {
                 .entity(Error.internalServerError(message))
     }
 
+    /**
+     * Returns a builder for an error response with multiple errors in the errors object
+     *
+     * Usage: compoundError([Error.notFound(), Error.forbidden()], 400).build()
+     *
+     * @param errorResultObjects is a list of non-compound ErrorResultObjects from the Error class
+     * @param status is the HTTP response code for the compound eror response. The developer
+     *      is responsible for supplying "the most generally applicable HTTP error code"
+     * @return compound error response builder
+     */
+    protected static ResponseBuilder compoundError(
+        List<ErrorResultObject> errorResultObjects, int status) {
+            def errors = []
+            for (int i = 0; i < errorResultObjects.size(); i++) {
```

Comment:
ErroResultObjects were passed because of the changes made to the skeleton when bringing it up to JSON API spec. In the Error class, the error constructors returned Error objects wrapped in an ErrorResultObject. 

This issue makes it clear that wrapping the Error object in an ErrorResultObject should happen in the ResponseBuilder for that error, so that Error objects can be manipulated before formatting for the response.

I've implemented these changes, preserving existing functionality.